### ISSUE TITLE
Adding folder to root / throws a jsp error

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#19934]
From the site browser, if you add a folder from the root directory, you get:

```
Feb 09, 2021 3:17:06 PM org.apache.catalina.core.ApplicationDispatcher invoke
SEVERE: Servlet.service() for servlet [jsp] threw exception
com.dotmarketing.business.DotStateException: Versionable is null
	at com.dotmarketing.business.IdentifierAPIImpl.find_aroundBody6(IdentifierAPIImpl.java:92)
	at com.dotmarketing.business.IdentifierAPIImpl$AjcClosure7.run(IdentifierAPIImpl.java:1)
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)
	at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:42)
	at com.dotcms.aspects.interceptors.CloseDBIfOpenedMethodInterceptor.invoke(CloseDBIfOpenedMethodInterceptor.java:29)
	at com.dotcms.aspects.aspectj.CloseDBIfOpenedAspect.invoke(CloseDBIfOpenedAspect.java:41)
	at com.dotmarketing.business.IdentifierAPIImpl.find(IdentifierAPIImpl.java:88)
	at org.apache.jsp.html.portlet.ext.folders.edit_005ffolder_jsp._jspService(edit_005ffolder_jsp.java:2189)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:71)
```

I would like to know what change caused this to happen?

**Comment by freddyucv** [comment]
**Note to QA:** really the bug happend when try to create a folder in any level (no just root) but the user have to be admin

**Comment by victoralfaro-dotcms** [comment]
Folder added correctly as an admin.
![image](https://user-images.githubusercontent.com/64026161/110548371-10607880-80f6-11eb-9231-f4ed1d102332.png)

**Comment by bryanboza** [comment]
Fixed, tested on release-21.03 // Postgres // FF
